package.json

```diff
@@ -133,6 +134,7 @@
     "eslint-config-airbnb": "^17.1.1",
     "eslint-import-resolver-webpack": "^0.11.1",
     "eslint-plugin-header": "^3.0.0",
+    "http-server": "^0.12.1",
```

Comment:
Is this a necessary dependancy for netlify?

Reply:
It is necessary in order to run tests during build. Test running is currently commented out in `netlifybuild.sh` since it was kind of hacky and seemed unstable but I left the dependancy just in case we want to re-enable it.